### PR TITLE
Integrated with Travis CI and Sonar. Resolved #3

### LINKED ISSUE #3: Integrate with Travis CI and Sonar.

### PR DESCRIPTION
**Pull request by mariuszkumor**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+language: java
+sudo: required
+dist: trusty
```
[the remark this answers is not in the document]
Comment (mariuszkumor) [reply .travis.yml]: OK

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+jdk:
+  - oraclejdk8
```
[the remark this answers is not in the document]
Comment (mariuszkumor) [reply .travis.yml]: OK, changed

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+language: java
+sudo: required
+dist: xenial
```
Comment (stlhrt) [inline .travis.yml]: did you check if it works, Travis docs say xenial is experimental and dockerized builds are better if you need more modern ubuntu available during build

Comment (mariuszkumor) [reply .travis.yml]: I did, look at the build status - it's green :-)

**Pull request merged**